web/templates/compare.html

```diff
@@ -58,15 +58,15 @@ <h5 class="text-center">{{ lang2_friendlyname }}'s Implementation</h5>
                 </div>
                 <div class="card">
                     <div class="card-body">
-                        <a href="https://github.com/codethesaurus/codethesaur.us/blob/master/web/thesauruses/{{ lang1 }}/{{ concept_id }}.json">
-                            Want to add or correct information for {{ lang1_friendlyname }}?
+                        <a href="https://github.com/codethesaurus/codethesaur.us/blob/master/web/thesauruses/{{ lang1|linebreaksbr }}/{{ concept_id|linebreaksbr }}.json">
```

Comment:
I would hope a <br> wouldn't ever break apart a URL! This should probably be removed. (Same for below, lines 68-69)